linkerd/core/src/main/scala/io/buoyant/linkerd/RequestActiveTracer.scala

```diff
@@ -0,0 +1,265 @@
+package io.buoyant.linkerd
+
+import com.twitter.finagle._
+import com.twitter.finagle.client.Transporter.EndpointAddr
+import com.twitter.finagle.http._
```

Comment:
I would expect an HTTP-specifc module like this to live in an HTTP-specific subproject (and not core). Can this be moved? If not, why not? How will we offer similar support for H2?